src/election.jl

```diff
@@ -172,17 +195,18 @@ end
 
 
 function ElectionTracker(election::Election,
-                         partisan_metrics::Array{S, 1}=AbstractScore[])::CompositeScore where {S <: AbstractScore}
+                         scores::Array{S, 1}=AbstractScore[])::CompositeScore where {S <: AbstractScore}
     """ The ElectionTracker method returns a CompositeScore that first updates
         the vote count / share for changed districts and then proceeds to
-        calculate other partisan metrics, as desired by the user.
+        run other scores (such as vote count for a particular party, partisan
+        metrics, etc.), as desired by the user.
         Re-calculating vote counts only for changed districts means that the
         CompositeScore does not perform redundant computations for all of the
-        partisan metrics. Furthermore, packaging all partisan metrics within
-        the CompositeScore ensures that the vote update occurs first, followed
-        by the partisan metrics scoring functions.
+        partisan metrics. Furthermore, packaging all election-related scores
+        within the CompositeScore ensures that the vote update occurs first,
+        followed by the partisan metrics scoring functions.
     """
-    vote_tracker = [count_votes("votes", election)]
```

Comment:
Previously, if you wanted to figure out the votes for Party A throughout the chain, you had to do `get_score_values(chain_data, "votes")` and then process an array of `Dict` objects...which makes no sense to the user because they never initialie a score with the name `"votes"` - that's something we do internally to this method!